src/libsystemd/sd-journal/journal-vacuum.c

```diff
@@ -48,6 +48,16 @@ static int vacuum_compare(const struct vacuum_info *a, const struct vacuum_info
         return strcmp(a->filename, b->filename);
 }
 
+static void vacuum_info_array_free(vacuum_info *list, size_t n) {
+        if (!list)
+                return;
```

Comment:
Maybe use an assert here?

```
assert(list || n == 0)
```

Reply:
The coding style is to exit early if the object is `NULL`.